### ISSUE TITLE
NVDA throws a lot of errors during opening IntelliJ IDEA

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#13039]
### Steps to reproduce:
1. Go to jetbrains.com/idea to download and install IntelliJ IDEA Community Edition 2021.2.3;
2. After installation, follow the default settings and create a new project arbitrarily;
3. Exit IntelliJ IDEA;
4. Open IntelliJ IDEA again, it will take a while to load, press Windows + M to minimize to the desktop during this time;

### Actual behavior:
During the execution of step (4), you will receive a large number of errors from NVDA when you minimize to the desktop. The log snippet is as follows:

```
ERROR - queueHandler.flushQueue (09:19:24.473) - MainThread (16392):
Error in func event_stateChange
Traceback (most recent call last):
  File "queueHandler.pyc", line 55, in flushQueue
  File "JABHandler.pyc", line 709, in event_stateChange
  File "NVDAObjects\__init__.pyc", line 85, in __call__
  File "NVDAObjects\JAB\__init__.pyc", line 231, in __init__
  File "NVDAObjects\window\__init__.pyc", line 167, in __init__
ValueError: invalid or not specified window handle
```

IntelliJ IDEA will be opened after you receive dozens of errors.
### Expected behavior:
NVDA has no error messages and IntelliJ IDEA should open smoothly,.

#### NVDA installed/portable/running from source:
installed and portable

#### NVDA version:
NVDA2021.3 Beta2

#### Windows version:
Windows 10 21H1 (x64) build 19043.1288

#### Name and version of other software in use when reproducing the issue:
IntelliJ IDEA Community Edition 2021.2.3

#### Other information about your system:

- CPU: Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz 1.90 GHz
- RAM: 16.0 GB
- Disk: 512 GB SSD

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The latest Alpha version can also be reproduced

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Brian1Gaff** [comment]
I guess this is a program written in or using Java then, is it?
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "Rowen" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Monday, November 08, 2021 1:54 AM
Subject: [nvaccess/nvda] NVDA throws a lot of errors during opening IntelliJ 
IDEA (Issue #13039)

**Comment by cary-rowen** [comment]
Yes, this is a program written in Java.

**Comment by cary-rowen** [comment]
@michaelDCurran @leonardder 
Do you have any thoughts on this?

**Comment by cary-rowen** [comment]
cc @seanbudd 
Could you please take a look at this issue?

**Comment by xingkong0113** [comment]
The same problem was encountered when using jetbrains

**Comment by mwhapples** [comment]
I have started taking a look at this. Something very important in reproducing which I did not initially appreciate, is needing to minimise the windows with Windows+M. If you do not minimise the windows then you do not get the errors.

**Comment by cary-rowen** [comment]
thanks @mwhapples
Yes, minimization is a necessary step.

**Comment by cary-rowen** [comment]
@mwhapples 
Thanks for your effort, I tested build #13796 and it worked fine for me, thanks again.
Hopefully this will be part of 2022.2 if possible.

**Comment by mwhapples** [comment]
That is great news. I suspect that this won't be included in 2022.2 because I think that has already been branched. Personally knowing the size of this change and its limited scope for causing unrelated issues, I would port it to the 2022.2 branch but I don't get to make that decission.
